models/issue.go

```diff
@@ -1874,3 +1874,96 @@ func UpdateReactionsMigrationsByType(gitServiceType structs.GitServiceType, orig
 		})
 	return err
 }
+
+// DeleteIssuesByIDs and Objects who depend on them
+func DeleteIssuesByIDs(ids []int64) (err error) {
+
+	sess := x.NewSession()
+	defer sess.Close()
+	if err = sess.Begin(); err != nil {
+		return err
+	}
+
+	deleteCond := builder.Select("id").From("issue").Where(builder.In("id", ids))
```

Comment:
If somebody knows a better way let me know!

Reply:
@lunny you are the xorm master any idear?

Reply:
I think we already have a issue delete function? Why we need the second?

Reply:
For a technical, we can define

```go
var deleteCond interface{} // []interface{} or *builder.Builder
```